### PR TITLE
Adds `share list file` (received files).

### PR DESCRIPTION
**Pull request by bonafidehan**
Usage: `share list file`.

PR on top of https://github.com/dropbox/dbxcli/pull/39.

### CODE REVIEW

File: cmd/share-list-files.go [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+	if err != nil {
+		return
+	}
+
+	printFiles(res.Entries)
+
+	for len(res.Cursor) > 0 {
+		continueArg := sharing.NewListFilesContinueArg(res.Cursor)
+
+		res, err = dbx.ListReceivedFilesContinue(continueArg)
```
Comment (diwakergupta) [inline cmd/share-list-files.go]: So is `share list files` supposed to enumerate files that have been shared with me ("received files") or files that I have shared as well? I would assume the latter, not knowing anything else about the API itself.

File: cmd/share-list-files.go [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+}
+
+func printFiles(entries []*sharing.SharedFileMetadata) {
+	for _, f := range entries {
+		fmt.Printf("%v\t%v\n", f.Name, f.PreviewUrl)
+	}
+}
+
+var shareListFilesCmd = &cobra.Command{
+	Use:   "received-file",
```
Comment (diwakergupta) [inline cmd/share-list-files.go]: Hmm, doesn't match naming conventions (e.g. file is called share-list-files). Update for consistency?

File: cmd/share.go [hunk 3]
```diff
@@ -23,6 +23,12 @@ var shareCmd = &cobra.Command{
 	Short: "Sharing commands",
 }
 
+var shareListCmd = &cobra.Command{
+	Use:   "list",
+	Short: "List shared things",
```
Comment (diwakergupta) [inline cmd/share.go]: s/things/"files, folders etc"?

**Comment by biounix** [comment]
hi

**Pull request closed without merging**